### PR TITLE
Add term entries for service messages

### PR DESCRIPTION
**Pull request by palango**
Add terminology entries for the service messages.

### CODE REVIEW

File: terminology.rst [hunk 1]
```diff
@@ -204,3 +204,12 @@ Terminology
...
         Specific block after which the lock in the :term:`locked transfer` expires
+
+   MonitoringRequest
+        A message that is broadcasted from a client to :term:`Monitoring Service` in order to provide the MS with the necessary information to watch the channel.
+
+   PFSCapacityUpdate
+        A message that is broadcasted from the client to the :term:`Pathfinding Service` in order to update the channel capacity.
+
+   PFSFeeUpdate
+        A message that is broadcasted from the client to the :term:`Pathfinding Service` in order to update the mediation fees.
```
Comment (konradkonrad) [inline terminology.rst]:
```suggestion
        A message that is broadcasted from the client to the :term:`Pathfinding Services<Pathfinding Service>` in order to update the mediation fees demanded by the client.
```

Use wording from docstring.

File: terminology.rst [hunk 2]
```diff
@@ -204,3 +204,12 @@ Terminology
 
    expiration
         Specific block after which the lock in the :term:`locked transfer` expires
+
+   MonitoringRequest
+        A message that is broadcasted from a client to :term:`Monitoring Service` in order to provide the MS with the necessary information to watch the channel.
+
+   PFSCapacityUpdate
+        A message that is broadcasted from the client to the :term:`Pathfinding Service` in order to update the channel capacity.
```
Comment (konradkonrad) [inline terminology.rst]:
```suggestion
        A message that is broadcasted from the client to the :term:`Pathfinding Services<Pathfinding Service>` in order to update the channel capacity.
```

File: terminology.rst [hunk 3]
```diff
@@ -204,3 +204,12 @@ Terminology
 
    expiration
         Specific block after which the lock in the :term:`locked transfer` expires
+
+   MonitoringRequest
+        A message that is broadcasted from a client to :term:`Monitoring Service` in order to provide the MS with the necessary information to watch the channel.
```
Comment (konradkonrad) [inline terminology.rst]:
```suggestion
        A message that is broadcasted from a client to :term:`Monitoring Services<Monitoring Service>` in order to request channel watching from the MS. The message contains all the necessary information for the MS to carry out a channel update for the user.
```

**Comment by palango** [comment]
Thanks for the fixes!

**Pull request merged**